app/controllers/repair_requests_controller.rb

```diff
@@ -44,6 +45,33 @@ def create
 
   private
 
+  def issue_dlo_work_orders
+    n_errors = 0
```

Comment:
is this number_of_errors?

Reply:
What do you mean? The variable is `n_errors` and it counts the number of API errors that happened trying to issue the work orders.

Reply:
I just mean the name was a little bit confusing when I first came across it

Reply:
do you want me to rename it?

Reply:
meh, it's probably ok

Reply:
then will you approve it? :upside_down_face: